plotextractor/converter.py

```diff
@@ -24,15 +24,14 @@
 
 """Functions related to conversion and untarring."""
 
-from __future__ import absolute_import, print_function
 
 import os
 import tarfile
 import re
 
 from time import time
 
-from subprocess32 import check_output, TimeoutExpired
+from subprocess32 import check_output
```

Comment:
```suggestion
from subprocess import check_output
```
subproess32 is a backport of python 3 subprocess to python 2.